src/main/java/com/waytta/SaltAPIBuilder.java

```diff
@@ -372,10 +375,12 @@ public static FormValidation doTestConnection(
             if (usedCredential == null) {
                 return FormValidation.error("CredentialId error: no credential found with given ID.");
             }
+            
+            Launcher launcher = Hudson.getInstance().createLauncher(TaskListener.NULL);
```

Comment:
`TaskListener.NULL` suppresses all log entries. It should go to the System log at least